<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>551</width>
    <height>673</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>293</width>
    <height>347</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>554</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>PMS通用参数配置</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>231</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string>IP配置</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>211</width>
       <height>22</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>数据库IP地址：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="edtDbIp"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>213</width>
       <height>22</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>本机IP地址　：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="edtHostIp"/>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>320</y>
      <width>231</width>
      <height>75</height>
     </rect>
    </property>
    <property name="title">
     <string>时间同步配置</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>43</y>
       <width>164</width>
       <height>22</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>时间同步间隔：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spTime">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="value">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>小时</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>20</y>
       <width>161</width>
       <height>16</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>充当服务器　：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chkServer">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>231</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>视频、车牌识别配置</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>153</width>
       <height>15</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>是否默认开启视频　　：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chkVideo">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>44</y>
       <width>153</width>
       <height>15</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>是否默认开启车牌识别：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chkPlate">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>68</y>
       <width>161</width>
       <height>16</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_13">
      <item>
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>车辆图片是否实时保存：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chkCarPic">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>610</y>
      <width>158</width>
      <height>25</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QPushButton" name="btnOk">
       <property name="text">
        <string>保存</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="text">
        <string>关闭</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>400</y>
      <width>231</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string>设置通道控制器地址与名称</string>
    </property>
    <widget class="QWidget" name="layoutWidget_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>201</width>
       <height>22</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_9">
      <item>
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>第二窗口：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="sp2">
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="maximum">
         <number>63</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="edt2">
        <property name="text">
         <string>出口视频</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>201</width>
       <height>22</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_10">
      <item>
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>第三窗口：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="sp3">
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="maximum">
         <number>63</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="edt3">
        <property name="text">
         <string>入口抓图</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>201</width>
       <height>22</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_11">
      <item>
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>第四窗口：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="sp4">
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="maximum">
         <number>63</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="edt4">
        <property name="text">
         <string>出口抓图</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>201</width>
       <height>22</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>第一窗口：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="sp1">
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="maximum">
         <number>63</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="edt1">
        <property name="text">
         <string>入口视频</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>410</y>
      <width>231</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>中心服务</string>
    </property>
    <widget class="QWidget" name="layoutWidget_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>211</width>
       <height>22</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_14">
      <item>
       <widget class="QLabel" name="label_14">
        <property name="text">
         <string>数据库IP地址  ：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="edtMsSqlIp"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>213</width>
       <height>22</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_15">
      <item>
       <widget class="QLabel" name="label_15">
        <property name="text">
         <string>数据接收IP地址：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="edtReceiverIp"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget_9">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>211</width>
       <height>22</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_18">
      <item>
       <widget class="QLabel" name="label_18">
        <property name="text">
         <string>FTP   IP地址  ：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="edtFtpIP"/>
      </item>
     </layout>
    </widget>
    <widget class="QCheckBox" name="chkAlert">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>201</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>显示报警信息</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_6">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>320</y>
      <width>231</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string>转发配置</string>
    </property>
    <widget class="QWidget" name="layoutWidget_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>211</width>
       <height>22</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_16">
      <item>
       <widget class="QLabel" name="label_16">
        <property name="text">
         <string>代理IP地址  ：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="edtAgentlIp"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>213</width>
       <height>22</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_17">
      <item>
       <widget class="QLabel" name="label_17">
        <property name="text">
         <string>是否连接中心：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chkToInternet">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_7">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>170</y>
      <width>231</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string>抓图时机</string>
    </property>
    <widget class="QWidget" name="layoutWidget_10">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>201</width>
       <height>22</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_19">
      <item>
       <widget class="QLabel" name="label_20">
        <property name="text">
         <string>第二窗口抓图时机：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cbxCapture2">
        <item>
         <property name="text">
          <string>圧地感</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>刷卡</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget_11">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>201</width>
       <height>22</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_20">
      <item>
       <widget class="QLabel" name="label_21">
        <property name="text">
         <string>第三窗口抓图时机：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cbxCapture3">
        <item>
         <property name="text">
          <string>圧地感</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>刷卡</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget_12">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>201</width>
       <height>22</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_21">
      <item>
       <widget class="QLabel" name="label_22">
        <property name="text">
         <string>第四窗口抓图时机：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cbxCapture4">
        <item>
         <property name="text">
          <string>圧地感</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>刷卡</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget_13">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>201</width>
       <height>22</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_22">
      <item>
       <widget class="QLabel" name="label_23">
        <property name="text">
         <string>第一窗口抓图时机：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cbxCapture1">
        <item>
         <property name="text">
          <string>圧地感</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>刷卡</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_8">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>10</y>
      <width>231</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string>其他</string>
    </property>
    <widget class="QCheckBox" name="chkOpenGate">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>43</y>
       <width>211</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>地感开闸</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="edtLedCan">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>20</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>第一个CAN地址为总空闲LED，
车位多个CAN地址用逗号分隔开即可。</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_19">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>独立显示屏CAN地址：</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="chkCapture">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>211</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>地感抓图按队方式列取图</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_24">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>79</y>
       <width>24</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>秒</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_25">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>79</y>
       <width>84</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>远距离卡识别间隔：</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spRemoteTime">
     <property name="geometry">
      <rect>
       <x>102</x>
       <y>79</y>
       <width>42</width>
       <height>20</height>
      </rect>
     </property>
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>10000</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_29">
     <property name="geometry">
      <rect>
       <x>13</x>
       <y>110</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>停车场ID</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="edtParkID">
     <property name="geometry">
      <rect>
       <x>78</x>
       <y>107</y>
       <width>141</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_9">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>210</y>
      <width>230</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>车牌实时显示</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>153</width>
       <height>16</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_12">
      <item>
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>第一路：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chkPlateReal1">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget_14">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>34</y>
       <width>153</width>
       <height>16</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_23">
      <item>
       <widget class="QLabel" name="label_26">
        <property name="text">
         <string>第二路：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chkPlateReal2">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget_15">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>57</y>
       <width>153</width>
       <height>16</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_24">
      <item>
       <widget class="QLabel" name="label_27">
        <property name="text">
         <string>第三路：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chkPlateReal3">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget_16">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>153</width>
       <height>16</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_25">
      <item>
       <widget class="QLabel" name="label_28">
        <property name="text">
         <string>第四路：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chkPlateReal4">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>551</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
